usage/keyboard-shortcuts.md

```diff
@@ -7,30 +7,31 @@ permalink: /usage/keyboard-shortcuts/
+| U                                               | Take a Screenshot of a video                                                                                                                                                                         |
+| Up Arrow                                        | Increase volume                                                                                                                                                                                      |
+| Down Arrow                                      | Decrease volume                                                                                                                                                                                      |
+| Left Arrow                                      | Rewind X seconds based on the Rewind Interval and Video Playback Rate Interval (Ex: 5 seconds Rewind Interval and Video Playback Rate Interval 1x, rewinds video 5 seconds)                          |
+| Right Arrow                                     | Fast-Forward X seconds based on the Fast-Forward Interval and Video Playback Rate Interval (Ex: 5 seconds Fast-Forward Interval and Video Playback Rate Interval 2x, fast-forwards video 10 seconds) |
+| 0-9                                             | Skip through video by percentage (Ex: 3 key to skip through 30% of a video)                                                                                                                          |
+| Alt + D or Command/Ctrl + L                     | Focus on the search bar                                                                                                                                                                              |
+| Play or Pause                                   | Play / Pause video                                                                                                                                                                                   |
+| Previous or Next                                | Skips to the Previous / Next video in a playlist                                                                                                                                                     |
+| , or .                                          | While the video is paused, skip to the previous / next frame.                                                                                                                                        |
+| SHIFT + Enter or SHIFT + Click on Search button | After user puts in search query use on of the following shortcuts to perform the search in a new instance                                                                                            |
```

Comment:
Could be a bit more concise
```suggestion
| SHIFT + Enter or SHIFT + Click on Search button | Search in a new instance of FreeTube                                                                                                                                         |
```

Reply:
Let me know your thoughts on this change

Reply:
Yes i completely agree. I didnt know how to properly word it.